src/uwtools/cli.py

```diff
@@ -42,17 +42,17 @@ def main() -> None:
     setup_logging(quiet=True)
     try:
         args, checks = _parse_args(sys.argv[1:])
-        for check in checks[args.mode][args.submode]:
+        for check in checks[vars(args)[STR.mode]][vars(args)[STR.submode]]:
```

Comment:
Replace references to `args` members where the expression uses the equivalent of an uncheckable string literal with a reference to the `STR` object, which can be verified by the linter. Many repetitions of this pattern follow.